src/resolver/resolve_path.zig

```diff
@@ -74,7 +74,13 @@ pub fn isParentOrEqual(parent_: []const u8, child: []const u8) ParentEqual {
     while (parent.len > 0 and isSepAny(parent[parent.len - 1])) {
         parent = parent[0 .. parent.len - 1];
     }
-    if (std.mem.indexOf(u8, child, parent) != 0) return .unrelated;
+
+    const contains = if (comptime !bun.Environment.isLinux)
```

Comment:
this is not technically correct

but it's probably okay enough for now.

case-sensitivity differs by filesystem - not by platform. it's just that on linux, most filesystems are case sensitive. on macos, the two most popular ones default to case insensitive, but HFS+ can be case sensitive if chosen. I don't know if Windows has an option for this.